platform/broadcom/saibcm-modules/debian/rules

```diff
@@ -67,6 +67,9 @@ kdist_clean: clean
 	    KERNDIR=/usr/src/linux-headers-$(KERNVERSION)-common \
 	    KERNEL_SRC=/usr/src/linux-headers-$(KERNVERSION)-amd64 \
 	    $(MAKE) -C systems/linux/user/x86-smp_generic_64-2_6 clean
+	SDK=$(realpath .) BUILD_KNET_CB=1 BUILD_PSAMPLE=1 \
```

Comment:
align the spacing

Reply:
![image](https://user-images.githubusercontent.com/10645050/106074241-57ebf180-60c0-11eb-8899-421694f4f2a4.png)

Not sure I understood. This is on par with the previous statement.

Reply:
you are right.